src/helix-ui/elements/HXElement.less

```diff
@@ -1,12 +1,11 @@
 @import (reference) "vars";
 
-*, *::before, *::after {
-  box-sizing: border-box;
-  color: inherit;
-  font: inherit;
-  letter-spacing: inherit;
-}
-
-input::-ms-clear {
-  display: none;
+:host {
+  *, *::before, *::after {
+    box-sizing: border-box;
+    // TODO: convert below properties into .inheritTypography() mixin
+    color: inherit;
+    font: inherit;
+    letter-spacing: inherit;
```

Comment:
We need to convert the `color`, `font`, and `letter-spacing` inheritance into a mixin for two reasons:

1. Any CSS that LESS generates in this file is applied to _every_ Shadow CSS (which has a linear impact on the file size of our JavaScript assets).
2. Not all elements with ShadowDOM need these styles applied. _I've not converted it to a mixin, because I don't know which elements need to consume the mixin explicitly._